src/common/aes/aes_ossl.c

```diff
@@ -65,15 +77,42 @@ void OQS_AES256_ECB_load_schedule(const uint8_t *key, void **schedule) {
 	EVP_CIPHER_CTX_set_padding(ks->ctx, 0);
 }
 
-void OQS_AES256_CTR_load_schedule(const uint8_t *key, void **schedule) {
+void OQS_AES256_CTR_inc_init(const uint8_t *key, void **schedule) {
 	*schedule = malloc(sizeof(struct key_schedule));
-	OQS_EXIT_IF_NULLPTR(*schedule);
 	struct key_schedule *ks = (struct key_schedule *) *schedule;
+	EVP_CIPHER_CTX *ctr_ctx = EVP_CIPHER_CTX_new();
+	assert(ctr_ctx != NULL);
+	//OQS_OPENSSL_GUARD(EVP_EncryptInit_ex(ctr_ctx, EVP_aes_256_ctr(), NULL, key, NULL));
```

Comment:
commented guard apparently initializing OpenSSL function intentional?

Reply:
yes, will remove.

Reply:
updated in 0b91245833c1bb9756519316fb13b8cb4d67d2cd